src/components/BackgroundLayout/index.js

```diff
@@ -1,6 +1,18 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+import defaultImage from 'images/auth_background.png';
+
+const BackgroundLayout = ({ dynamicImage, noBackground, children }) => {
```

Comment:
why do we need an additional `noBackground` prop here? I think we can omit the `dynamicImage` image on the component which doesn't have the background? instead of introducing a new prop `noBackground`? What do you think?

Reply:
Updated in [23898e2](https://github.com/rafayet-monon/surveyor/pull/12/commits/23898e241599d9c0aa71c074d9e36bc9c0dc66fd). Removed the `noBackground` prop.